docs/post-cloner.md

```diff
@@ -0,0 +1,71 @@
+# Post Cloner
+
+This feature adds a button to the post edit screen that allows a user to clone a post with its metadata and terms to a new, identical post.
+
+It is enabled by default but can be disabled via the configuration file:
+
+```json
+{
+	"extra": {
+		"altis": {
+			"modules": {
+				"cms": {
+					"cloner": false
+				}
+			}
+		}
+	}
+}
+```
+
+## Actions
+
+**`post_cloner_after_meta_data_copy: (int) $post_id`**
+
+Fires when post meta has been copied to cloned post.
+
+## Filters
+
+**`post_cloner_cloned_parent: (int) $original_post['ID']`**
+
+Allows overriding the parent that the new post should have.
+
+**`post_cloner_cloned_status: (string) 'draft'`**
+
+Allows overriding the status that the new post should have
+
+**`post_cloner_name_append: (string) '-cloned'`**
+
+Allows overriding the string to append onto the end of post name to prevent collisions
+
+**`post_cloner_meta_keys_to_remove: (array) $keys_to_remove`**
+
+Allows modifying the denylist of keys of metadata that should not be passed through to the new post.
+
+**`post_cloner_meta_patterns_to_remove: (array) []`**
+
+Allows overriding the regex patterns to look for when cleaning meta keys
```

Comment:
Can we explain this one a little more clearly, perhaps with an example value? I'm not sure what this means exactly, but guessing it's related to the filter above.

Reply:
I'm not sure either, I suppose it's if you want to skip copying all meta data with a key like `foo_`

Reply:
Could you look into how it's used in the post-cloner code so we can be sure?